intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -822,560 +822,3 @@ \section{Transfer Files to/from the \hpc}
 \subsection{Fast file transfer for large datasets}
 
 See \href{\LinuxManualURL#sec:rsync}{the section on \texttt{rsync} in chapter 5 of the Linux intro manual}.
-
-\section{Modules}
-\label{sec:modules}
```

Comment:
Any references that got broken?

Can we easily detect broken references with a `grep` in Travis (if so, should be done in a separate PR, not here)

Reply:
References didn't get broken, since I didn't remove any labels (just moved them).

Reply:
OK, but please open an issue to add a check in Travis to check for broken references, I don't think that makes the LaTeX build fail, does it?

Reply:
Done, see #295